### ISSUE TITLE
problem about result

### ISSUE THREAD

**Issue by 1529046970** [Timforce/tiny-tf#2]
Hi,I trained the model myself,but my evaluation result in medium set is 0.821. What's wrong with me? Thank you!
![WechatIMG4](https://user-images.githubusercontent.com/40976076/54577035-2a01e480-4a35-11e9-965a-46645b17e6fc.jpeg)

**Comment by Timforce** [comment]
Hi,
Although this implementation can get similar result to the original paper, it can be unstable to train due to the arithmetic of the loss value. I will look into this and perhaps try to fix the unstable training issue. 

Here are a couple of advice that may help you for now:
1. Give it more time(epochs) to train, evaluate each epoch checkpoints and pick the best from it.
2. Tweak the [SAMPLE_LIMIT][url] in cfg files and not by too much, range between 0~64 should be fine. 

[url]:https://github.com/Timforce/tiny-tf/blob/master/cfgs/tiny_resnet101.yml#L21

**Comment by 1529046970** [comment]
Thanks for your reply! I tried to use the pretrained model you provided, but the result in medium set is also 0.821. Can you give me some advice, is it that I made a mistake at some step?

**Comment by 1529046970** [comment]
I tried the model you provided and the model I trained myself, both of the results are 0.821.

**Comment by Timforce** [comment]
Hi, 
My guess is that you may not have run through all the validation image set. 
To get a proper measurement, you will need to run detection through all 3226 images and get the corresponding txt files. To checkout the txt files number within 'pred' directory:
```Shell
find pred -name "*.txt" | wc -l
```
If the number does not match, it's probably because the network ran out of GPU memory in the middle of the detection, which also stopped the detection of the following validation set images.

As for the same result between the pre-trained model and your own, i'm not entirely sure how does that happened. Perhaps you should double check if the --weights path of the [shell script](https://github.com/Timforce/tiny-tf/blob/master/tiny_resnet101_eval.sh) is pointing to the correct checkpoint path.

i hope these info may help you :)

**Comment by 1529046970** [comment]
Thank you very much for your reply! Sorry to bother you again! I tested as you suggested, and the images are 3226! 
![IMG_2680](https://user-images.githubusercontent.com/40976076/54877335-f6212780-4e57-11e9-8c86-f2f0a9525aa2.JPG)
And I check the predict folder , files are as follow
![捕获](https://user-images.githubusercontent.com/40976076/54877347-1e108b00-4e58-11e9-9c23-894eca7bc684.JPG)

**Comment by 1529046970** [comment]
I get the result as you showed! There ie something wrong with my computer. Thank you very much again!

**Comment by Timforce** [comment]
Hi, glad it works fine now!
If it is a big issue that may cause the evaluation tool keep generating inconsistent result, please let me know or share how to fix it :)

**Comment by 1529046970** [comment]
I don't know what happened. I just tried on another computer, then I get the right result.  Thank you again 👍  :)
